natural-language-understanding/src/main/java/com/ibm/watson/developer_cloud/natural_language_understanding/v1/model/AnalysisResults.java

```diff
@@ -0,0 +1,343 @@
+
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * Results of the analysis, organized by feature.
+ */
+public class AnalysisResults extends GenericModel {
+
+  /** Language used to analyze the text. */
+  private String language;
+  /** Text that was used in the analysis. */
+  @SerializedName("analyzed_text")
+  private String analyzedText;
+  /** URL that was used to retrieve HTML content. */
+  @SerializedName("retrieved_url")
+  private String retrievedUrl;
+  /** API usage information for the request. */
+  private Usage usage;
+  /** The general concepts referenced or alluded to in the specified content. */
+  private List<ConceptsResult> concepts;
+  /** The important entities in the specified content. */
+  private List<EntitiesResult> entities;
+  /** The important keywords in content organized by relevance. */
+  private List<KeywordsResult> keywords;
+  /** The hierarchical 5-level taxonomy the content is categorized into. */
+  private List<CategoriesResult> categories;
+  /** The anger, disgust, fear, joy, or sadness conveyed by the content. */
+  private EmotionResult emotion;
+  /** The metadata holds author information, publication date and the title of the text/HTML content. */
+  private MetadataResult metadata;
+  /** The relationships between entities in the content. */
+  private List<RelationsResult> relations;
+  /** The subjects of actions and the objects the actions act upon. */
+  @SerializedName("semantic_roles")
+  private List<SemanticRolesResult> semanticRoles;
+  /** The sentiment of the content. */
+  private SentimentResult sentiment;
+
+  /**
+   * Adds the concepts.
+   *
+   * @param concepts the new concepts
+   */
+  public void addconcepts(ConceptsResult concepts) {
+    this.concepts.add(concepts);
+  }
+
+  /**
+   * Adds the entities.
+   *
+   * @param entities the new entities
+   */
+  public void addentities(EntitiesResult entities) {
+    this.entities.add(entities);
+  }
+
+  /**
+   * Adds the keywords.
+   *
+   * @param keywords the new keywords
+   */
+  public void addkeywords(KeywordsResult keywords) {
+    this.keywords.add(keywords);
+  }
+
+  /**
+   * Adds the categories.
+   *
+   * @param categories the new categories
+   */
+  public void addcategories(CategoriesResult categories) {
+    this.categories.add(categories);
+  }
+
+  /**
+   * Adds the relations.
+   *
+   * @param relations the new relations
+   */
+  public void addrelations(RelationsResult relations) {
+    this.relations.add(relations);
+  }
+
+  /**
+   * Adds the semanticRoles.
+   *
+   * @param semanticRoles the new semanticRoles
+   */
+  public void addsemanticRoles(SemanticRolesResult semanticRoles) {
+    this.semanticRoles.add(semanticRoles);
+  }
+
+  /**
+   * Gets the language.
+   *
+   * @return the language
+   */
+  public String getLanguage() {
```

Comment:
I think the responses can be plain POJO because we don't need them to follow the same pattern as the Request objects. Responses are not implementing the Builder pattern